nilearn/plotting/tests/test_matrix_plotting.py

```diff
@@ -12,13 +12,14 @@
 def test_matrix_plotting():
     from numpy import zeros
     mat = zeros((10, 10))
-    labels = str(range(10))
+    labels = [str(i) for i in range(10)]
     ax = plot_matrix(mat, labels=labels, title='foo')
     plt.close()
     # test if plotting lower triangle works
     ax = plot_matrix(mat, labels=labels, tri='lower')
     # test if it returns an AxesImage
     ax.axes.set_title('Title')
     plt.close()
-
-
+    # test if reordering with default linkage works
+    ax = plot_matrix(mat, labels=labels, reorder=True)
```

Comment:
I think we should have a more specific test checking whether the reordering is sensible.

Reply:
I built now a matrix with some similarities and check if the block structure is present in the reordered labels of the returned AxesImage. A bit convoluted, but it checks if the clustering worked.